locale/en_US/locale.xml

```diff
@@ -101,6 +101,7 @@
 	<message key="section.title">Section title</message>
 	<message key="section.abbreviation">Abbreviation</message>
 	<message key="section.abbreviation.example">(For example, Articles=ART)</message>
+	<message key="section.policy">Section Policy</message>
```

Comment:
Does this also require a PR for OMP?

Reply:
> Does this also require a PR for OMP?

I don't think so as the change is part of pkp/ojs and is used in `sectionPolicy.tpl` which is also part of pkp/ojs only.